keyboards/mechlovin/olly/jf/rev2/rev2.c

```diff
@@ -0,0 +1,122 @@
+  {0,51},     {12, 51},      {24, 51},      {37, 51},      {49, 51},      {62, 51},      {74, 51},       {87,51},       {99,51},       {112,51},      {124,51},      {137,51},      {149,51},      {162,51},      {174,51},      {186,51},       {199,51},       {211,51},       {224,51},
+  {0,64},     {12, 64},                     {37, 64},                                    {74, 64},                                                    {124,64},      {137,64},      {149,64},      {162,64},      {174,64},      {186,64},       {199,64},       {211,64},       {224,64},
+    }, {
+       1,                           1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,
+       1,            1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,
+       1,            1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,
+       1,            1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,                            1,             1,             1,             1,
+       1,            1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,             1,
+       1,            1,                            1,                                           1,                                                          1,             1,             1,             1,             1,             1,             1,             1,             1,
+    } 
+};
+
+
+bool led_matrix_indicators_kb(void) {
+    if (!led_matrix_indicators_user()) { return false; }
+    if (host_keyboard_led_state().caps_lock) {
+        led_matrix_set_value(106, 0xFF);
+        led_matrix_set_value(56, 0xFF);
+    } else {
+        led_matrix_set_value(106, 0x00);
+    }
+    if (host_keyboard_led_state().num_lock) {
+        led_matrix_set_value(107, 0xFF);
+        led_matrix_set_value(33, 0xFF);
+    } else {
+        led_matrix_set_value(107, 0x00);
+    }
+    if (host_keyboard_led_state().scroll_lock) {
+        led_matrix_set_value(108, 0xFF);
+    } else {
+        led_matrix_set_value(108, 0x00);
+    }
+    return true;
+}
+
+
+layer_state_t layer_state_set_kb(layer_state_t state) {
```

Comment:
The code here should also be in led_matrix_indicators_kb, as well. 

If there is an animation enabled, these values would get wiped out.

Reply:
okay.